### ISSUE TITLE
Mods dont work on multiplayer

### ISSUE THREAD

**Issue by Creeperface01** [zhuowei/MCPELauncher#251]
I Want to make mod compatibile with my pocketmine plugin.  So please fix it.

**Comment by sliceofcode** [comment]
@Creeperface01 Mods are not intended for Multiplayer. I highly doubt @zhuowei will add support for them.

**Comment by RandomAltThing** [comment]
They work on multiplayer somehow (maybe its patches) but people can still hack on servers -_-. Its hard to be an admin when you have to to to all invisible hackers.

**Comment by Creeperface01** [comment]
I think my players dont know about this hack xD

**Comment by connerfick** [comment]
Can someone please ask the person who made too many items to make a version
for Pocketmine?

**Comment by Creeperface01** [comment]
What? :D too Many items for pocketmine? It is The same like TMI for craftbukkit :DDDDD

**Comment by connerfick** [comment]
@creeperface01 yes like tmi from mcpc.

**Comment by Creeperface01** [comment]
When you put TMI or NEI to MCPC plugin Folder i think it wont work And you must have plugin for pocketmine for enable TMI

**Comment by RandomAltThing** [comment]
Too many items on servers! AWESOME IDEA @McMrArm . If you are on a server and select an item, it would do the command /give ! If you dont have perms then too bad but still

**Comment by Creeperface01** [comment]
Yes it would be good but zhuowei still can not make BL for my tab —,—

**Comment by RandomAltThing** [comment]
xD

**Comment by KairusDarkSeeker** [comment]
Zhuowei and the Server Staffs have decided that mods do not work in MCPE 0.11.x
